hare/hare.go

```diff
@@ -119,23 +119,31 @@ func New(conf config.Config, p2p NetworkService, sign Signer, nid types.NodeId,
 
 // checks if the provided id is too late/old to be requested.
 func (h *Hare) isTooLate(id instanceId) bool {
```

Comment:
we can name this better... e.g. includeInOutputBuffer

Reply:
currently, the new impl doesn't check if it is actually inside the buffer, but checks if the given id is more than `bufferSize` in the past.

Reply:
I meant _(should we) include in hare output buffer_ so the returned answer should be true/false
But other names are welcome. `IsTooLate` has no context.

Reply:
how about, `outOfHdist` ?

Reply:
it is not really correlated to hdist at the moment.